chalice/app.py

```diff
@@ -0,0 +1,147 @@
+import json
+import traceback
+
+from chalice import Chalice, NotFoundError, BadRequestError
+from cloud_blobstore import BlobNotFoundError
+from chalicelib.config import logger
+from chalicelib.matrix_handler import LoomMatrixHandler
+from chalicelib.request_handler import RequestHandler, RequestStatus
+from chalicelib.sqs import SqsQueueHandler
+
+app = Chalice(app_name='matrix-service-api')
+app.debug = True
+
+# Replace handler here for supporting concatenation on other matrix formats
+mtx_handler = LoomMatrixHandler()
+
+
+@app.route('/matrices/health', methods=['GET'])
+def health():
+    return {'status': 'OK'}
+
+
+@app.route('/matrices/concat/{request_id}', methods=['GET'])
+def check_request_status(request_id):
+    """
+    Check the status of a matrices concatenation request based on
+    a request ID.
+
+    :param request_id: Matrices concatenation request ID.
+    """
+    try:
+        merge_request_body = RequestHandler.get_request_body(request_id=request_id)
+        return merge_request_body
+
+    except BlobNotFoundError:
+        raise NotFoundError("Request({}) has not been initialized.".format(request_id))
+
+    except:
+        raise BadRequestError(traceback.format_exc())
+
+
+@app.route('/matrices/concat', methods=['POST'])
+def concat_matrices():
+    """
+    Concat matrices within bundles based on bundles uuid
+    """
+    request = app.current_request
+    bundle_uuids = request.json_body
+    request_id = RequestHandler.generate_request_id(bundle_uuids)
+
+    logger.info("Request ID({}): Received request for concatenating matrices within bundles {};"
```

Comment:
python 3.6+?  you can use f-style strings.

Reply:
I will switch to use f-style strings.